src/main/java/org/spongepowered/api/entity/living/player/Player.java

```diff
@@ -294,4 +296,22 @@ default GameModeData getGameModeData() {
      */
     boolean respawnPlayer();
 
+    /**
+     * Gets this {@link Player}'s real textures property.
+     *
+     * @return The real textures property
+     */
+    Optional<ProfileProperty> getRealTextures();
```

Comment:
I don't like this being called get**Real**Textures. All the rest of the naming is either unmarked (for 'real' things) or prepended with `Fake` for the fake things.

Reply:
Can we introduce a ' global context' and make context specifying explicit for things that would potentially need it? I prefer 'real' naming to leaving it off, but would prefer something that was either provides a 'global context' or was named as such.